templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails server Puma -p ${PORT:-3000} -c ./config/puma.rb
```

Comment:
Why default to `3000`? That seems like an additional (unnecessary) change. Is it something to do with Puma? Doesn't foreman handle `$PORT` for us?